graphql_client_cli/src/introspect_schema.rs

```diff
@@ -20,11 +19,11 @@ pub fn introspect_schema(
     authorization: Option<String>,
     headers: Vec<Header>,
     no_ssl: bool,
-) -> anyhow::Result<()> {
+) {
     use std::io::Write;
 
     let out: Box<dyn Write> = match output {
-        Some(path) => Box::new(::std::fs::File::create(path)?),
+        Some(path) => Box::new(::std::fs::File::create(path).unwrap()),
```

Comment:
Not sure I like dropping decent error handling along with `anyhow`, but I prefer better error types even in executables, so…

Reply:
The problem of error handling here is mostly about displaying useful context when something fails (nothing is handled). In my experience, anyhow isn't much better at this than panics. I'd be ok reverting the anyhow drop for the CLI. For reference, this is the error type I work with most of the time: https://github.com/prisma/prisma-engines/blob/master/migration-engine/connectors/migration-connector/src/error.rs — it's evolved into its current form over time, and it's exactly the right thing for error reporting in prisma.

Reply:
Working on a quick error type for graphql_client_cli

Reply:
Pushed.